tuf/sig.py

```diff
@@ -377,3 +367,106 @@ def generate_rsa_signature(signed, rsakey_dict):
   signature = tuf.keys.create_signature(rsakey_dict, signed)
 
   return signature
+
+
+
+
+
+def sign_over_metadata(
+    key_dict, data,
+    metadata_format=tuf.conf.METADATA_FORMAT):
+  """
+  Higher level function that wraps tuf.keys.create_signature, and works
```

Comment:
We should also keep up the habit of updating the units tests and maintaining code coverage.  The pull request to upstream will fail if code coverage is less than a certain percent.